src/main/java/mil/dds/anet/beans/Task.java

```diff
@@ -202,12 +204,21 @@ public Organization getResponsibleOrg() {
   public CompletableFuture<AnetBeanList<Report>> loadReports(
       @GraphQLRootContext Map<String, Object> context,
       @GraphQLArgument(name = "query") ReportSearchQuery query) {
-    // TODO: Use the query parameter
     if (reports != null) {
       return CompletableFuture.completedFuture(reports);
```

Comment:
If we still want to cache here, we need to cache per query

Reply:
Good point. Judging by the coverage report, caching isn't useful here (and we should be able to rely on the caching done by the batcher in this case anyway). Easier to remove this local cache altogether.